src/main/java/org/kainos/ea/cli/EmployeeRequest.java

```diff
@@ -0,0 +1,64 @@
+package org.kainos.ea.cli;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class EmployeeRequest {
+    private String f_name;
```

Comment:
These aren't the standard Java format for variables

Reply:
That should be all fixed now!